package.json

```diff
@@ -174,9 +177,9 @@
     "yup": "^0.32.0"
   },
   "dependencies": {
+    "@fastify/ajv-compiler": "^1.0.0",
     "@fastify/proxy-addr": "^3.0.0",
     "abstract-logging": "^2.0.0",
-    "ajv": "^6.12.2",
```

Comment:
this is still needed for the compiled options runtime.

Reply:
Do you mean the `build/build-validation.js` script?
Is not it created with dev dependencies on a local env?

Reply:
The result still depends on ajv at runtime.

Reply:
I think I'm missing something here:
the `configValidator.js` script is built from the `build-validation.js` to avoid the ajv loading when the user creates a new fastify instance.
The `build-validation.js` script is executed offline and without any automation and it is not executed when the user install the `fastify` module.
For these reasons I thought that move `ajv` to the devDependency was fine

Where am I wrong on this workflow?

Reply:
The generated script has a runtime dependency on some utilities within Ajv.

Reply:
Sorry to bother you but I can't find any `ajv` reference excepts in `/test` and `build-validation.js`

I have run `rm -rf node_modules/ && npm install --production` and run a server with input parameters and it works

```
const fastify = require('./')({ keepAliveTimeout: 'not integer' })
fastify.listen(8080)
```

Reply:
This is ok. However we had https://github.com/fastify/light-my-request/pull/115#issuecomment-747727147 in the past. Can you check if it'd also be the same for Ajv@8?

Reply:
No, i have checked: the output of the `4.4.0` that has the issue produce the `validate.js` that contains this line:

```
...
const func0 = require("ajv/dist/compile/equal").
...
```

While the output of the `build-validation.js` has not this statement